src/schemas/21-526EZ-allclaims/schema.js

```diff
@@ -34,6 +17,14 @@ const serviceBranches = [
   'Public Health Service'
 ];
 
+const specialIssues = {
```

Comment:
Can we define this in the schema's `definitions` only?

Reply:
ahh, nice catch